lib/flat_search/filters.ex

```diff
@@ -5,6 +5,5 @@ defmodule FlatSearch.Filters do
 
   alias FlatSearch.Filters.Filter
 
-  def change_filter(%Filter{} = filter), do: Filter.changeset(filter, %{})
-  def change_filter(%Filter{} = filter, _), do: Filter.changeset(filter, %{})
+  def change_filter(%Filter{} = filter, params \\ %{}), do: Filter.changeset(filter, %{})
```

Comment:
then you can pass the params variable to the changeset function instead of empty map

Reply:
since this would be an empty map as default when not passed

Reply:
yeah, right